go.mod

```diff
@@ -5,5 +5,9 @@ go 1.15
 require (
 	github.com/SKF/go-rest-utility v0.2.0
 	github.com/SKF/go-utility/v2 v2.12.0
-	github.com/stretchr/testify v1.6.1
+	github.com/go-openapi/errors v0.20.1
+	github.com/go-openapi/strfmt v0.21.0
+	github.com/go-openapi/swag v0.19.15
+	github.com/go-openapi/validate v0.20.3
```

Comment:
No new imports, dont think we need these?

Reply:
They're needed by the generated code, so if code is built and you run `go mod tidy` they're added. Not sure if we should leave them out?

Reply:
Looks like the build fails if I remove them.

Reply:
Then I was wrong, strange that it built before.